lib/new_file.rb

```diff
@@ -0,0 +1,5 @@
+class Hoge
+   # A comment with wrong indentation
+
+   # A comment with wrong indentation
```

Comment:
<article><p><b>[Rubocop]</b> <span>Incorrect indentation detected (column 3 instead of 2).</span></p><details><p></p><p>4 line , 4 column in <code>lib/new_file.rb</code></p><dl><dt>Rule</dt><dd>Layout/CommentIndentation</dd><dt>Severity</dt><dd>convention</dd></dl></details></article>